omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1884 @@
+********************
+Version requirements
+********************
+
+Proposal for revised minimum version requirements
+-------------------------------------------------
+
+Minimum software versions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is a proposal to the wider community to solicit their
+feedback. For each component, we need to decide upon the minumum
+supportable version which is provided by a distribution and/or is
+installable independently. Criteria for what is considered to be
+supportable includes support by its developers upstream and within
```

Comment:
I don't know if a "both" after "support" would make this sentence easier to parse. I certainly had to re-read it a couple of times.